src/configuration_parameters1.rs

```diff
@@ -0,0 +1,66 @@
+/*
+ *   Copyright (c) 2020 
+ *   All rights reserved.
+ */
+use clap::{App, Arg};
+
+pub fn get_configuration_parameters(app_name: &str) -> ConfigurationParameters {
+    let matches = get_eligible_arguments_for_app(app_name);
+    ConfigurationParameters::new_from_matches(matches)
+}
+
+pub struct ConfigurationParameters {
+    pub input_file_path: String,
+    pub output_file_path: String,
+
+}
+
+impl ConfigurationParameters {
+fn new_from_matches(matches: clap::ArgMatches) -> ConfigurationParameters {
+    let input_file_path = matches
+        .value_of("input_file_path")
+        .expect("Error getting `input_file_path` value.")
+        .to_string();
+
+
+        let output_file_path = matches
+        .value_of("input_file_path")
+        .expect("Error getting `output_file_path` value.")
+        .to_string();
+
+        ConfigurationParameters {
+            input_file_path,
+            output_file_path,
+        }
+}
+}
+fn get_eligible_arguments_for_app(app_name: &str) -> clap::ArgMatches {
+    App::new(app_name)
+        .about("This app modifies data to conform with the input requirements of BILLS CFGen!")
+        .arg(
+            Arg::with_name("input_file_path")
+                .long("input_file")
+                .short("i")
+                .help("Path to input file that needs to be processed.")
+                .allow_hyphen_values(true)
```

Comment:
why u have choosen to add this option here.....also why only here and not in output file argument